### PR TITLE
Fejer2 algorithm for integration

### PR DESCRIPTION
**Pull request by m-balesdent**
Proposal of Fejer2 class for the multi-dimensional integration using Fejer2 nodes and weights formula.

### CODE REVIEW

**Comment by jschueller** [comment]
great! I have only minor comments

File: python/src/Fejer2_doc.i.in [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+Examples
+--------
+>>> import openturns as ot
+>>> f = ot.SymbolicFunction(['x'], ['sin(x)'])
+>>> a = -2.5
+>>> b = 4.5
+>>> algoF2 = ot.Fejer2([10])
+>>> value = algoF2.integrate(f, ot.Interval(a, b))[0]
+>>> print(value)
+-0.590...
```
Comment (jschueller) [inline python/src/Fejer2_doc.i.in]: almost there, here the result is -0.591

Comment (m-balesdent) [reply python/src/Fejer2_doc.i.in]: It's a problem on doc, I gonna modify it.

**Comment by jschueller** [comment]
@m-balesdent Good for me, it just needs the ChangeLog to be updated to mention the new class, and you can add yourself to the list of authors in the about.rst file.

**Comment by jschueller** [comment]
someone knows who was assigned to the Fejer1 version ?

**Comment by m-balesdent** [comment]
It's Loïc

**Comment by jschueller** [comment]
can he push something on github, even if not finished ?

**Comment by jschueller** [comment]
moved to #1548, thanks!

**Pull request closed without merging**